tests/main/views/test_drafts.py

```diff
@@ -28,6 +31,9 @@ def setup(self):
             'lot': 'scs',
             'supplierId': 1
         }
+        self.basic_questions_to_copy = {
+            'questionsToCopy': ['serviceName']
```

Comment:
I think the name of this variable implies the value will just be a list rather than a dict to be unpacked into a method call. Probably this should just be `['serviceName']`

Reply:
I think it's nice having this defined like this, as it saves the boilerplate of writing out the containing dict for the list. I take your point about the name though. I'll change it to be more explicit. Something like `basic_questions_json` to be consistent with the other fixtures defined just above.